### ISSUE TITLE
Enable Curation of Organizational Affiliation List

### ISSUE THREAD

**Issue by sgoggins** [chaoss/metrics#99]
There is a broad need for the curation of a list of email addresses and organizational affiliations. Curation of such a list will be helpful for developing more precise implementations of metrics related to organizational contribution levels.  
Questions of privacy associated with the new European data laws must also be addressed and considered.  One possibility is a Mozilla project aimed at supporting individual, global consent levels across sites we may use, like GitHub.

**Comment by jgbarah** [comment]
Good point, @sgoggins. In GrimoireLab we're using [SortingHat](https://github.com/chaoss/grimoirelab-sortinghat) for that purpose. It provides a Python API to an SQL database where you can store and curate information related to identities (not only email addresses) of developers, affiliations, and that sort of stuff. Then, it can be queried by other tools to retrieve, eg, all identities for a developer, or the affiliation of a certain developer.

I agree that GDPR may apply, depending on the context. When you're talking about "a Mozilla project", is that some project in particular, or maybe something that they may do?

**Comment by GeorgLink** [comment]
Organizational Affiliation List:
- Something like ORCID.org for developers?
- Or a Keybase.io account?
- Or an enhanced GitHub account with additional links to other identities?

**Comment by germonprez** [comment]
What is the status of this? Not being implemented? Can this issue be closed?

**Comment by sgoggins** [comment]
@germonprez : The prevalence of tools cannot be confused with this being a solved problem at this time. There remains a high need and limited trust for shared maintenance of something like this right now.

**Comment by germonprez** [comment]
Not suggesting it's solved, just wondering the status. No problem if it stays open
